src/ray/raylet/dependency_manager.h

```diff
@@ -245,6 +298,10 @@ class DependencyManager : public TaskDependencyManagerInterface {
   /// tasks are ready to run and which `ray.wait` requests can be finished.
   std::unordered_set<ray::ObjectID> local_objects_;
 
+  /// Counts the number of active task dependency fetches by task name. The counter
+  /// total will be less than or equal to the size of queued_task_requests_.
```

Comment:
Maybe waiting_task_counter_ is a better name? Active requests and active fetches is a bit confusing since these tasks may not actually match up with the objects actively fetched.